js/services/MeteorService.js

```diff
@@ -0,0 +1,40 @@
+const dateUtils = require('../utils/DateUtils.js')
+const requestUtils = require('../clients/MeteorClient.js')
+const meteorMapper = require('../mappers/MeteorMapper.js')
+
+const monday = dateUtils.getDayOfTheCurrentWeek(1)
```

Comment:
What does 1 & 5 mean? It not obvious, so please avoid using anonymous values, try to add sense to them (e.g. mondayOffset or smth else)

Reply:
Thanks!
Replaced with date-fns